<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusChangePassword</class>
 <widget class="QDialog" name="MotusChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>449</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>486</width>
    <height>449</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>486</width>
    <height>449</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MotusChangePassword{border-image: url(:/picture/passwordChange.png);}</string>
  </property>
  <widget class="QPushButton" name="trueButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>271</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(147,199,151);} 
QPushButton:hover{background-color: rgb(131,189,131);} QPushButton:pressed{padding-left:3px;
padding-top:3px;background-color: rgb(78,182,171);}
QPushButton{border-radius:5px;}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newpasswordlineEdit">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>197</y>
     <width>206</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0;border-style:outset;background:transparent;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="againpasswordlineEdit">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>249</y>
     <width>192</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0;border-style:outset;background:transparent;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldpasswordlineEdit">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>145</y>
     <width>206</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0;border-style:outset;background:transparent;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
